protobufs/livekit_rtc.proto

```diff
@@ -119,16 +119,16 @@ message SimulcastCodec {
   string codec = 1;
   string cid = 2;
   repeated VideoLayer layers = 4;
+  VideoLayer.Mode video_layer_mode = 5;
 
-  // NEXT-ID: 5
+  // NEXT-ID: 6
 }
 
 message AddTrackRequest {
   // client ID of track, to match it when RTC track is received
   string cid = 1;
   string name = 2;
   TrackType type = 3;
-  // to be deprecated in favor of layers
```

Comment:
removing this comment as this is used in adaptive streaming and also useful to have this as capture size or max size. The width and height are also in `UpdateLocalVideoTrack` message. So, this comment is misleading.